src/main/resources/application.properties

```diff
@@ -80,4 +80,4 @@ google.defaultFirestoreLocation=us-central
 google.firestoreRetries=10
 management.health.probes.enabled=true
 management.endpoints.web.exposure.include=*
-features.search.api=disabled
+features.search.api=enabled
```

Comment:
I don't think we'll want to commit this when we merge, we'll want to overwrite the env var in dev.

Reply:
yeah this was an accident